cli/src/pcluster/config/imagebuilder_config.py

```diff
@@ -226,7 +226,7 @@ def __init__(
         config_region: str = None,
         custom_s3_bucket: str = None,
         source_config: str = None,
-        deployment_settings: DeploymentSettings = None,
+        deployment_settings: BaseDeploymentSettings = None,
```

Comment:
Even though the imagebuilder deployment settings are the base right now, since we are splitting this into two, we should probably create the image builder deployment settings class wrapper now

Reply:
I think this could be a future change. As there are not fields that are separately added for ImageBuilder which is why I have kept BaseDeploymentSettings